client/modules/i18n/main.js

```diff
@@ -119,7 +119,7 @@ Meteor.startup(() => {
           locale.language = getBrowserLanguage();
           moment.locale(locale.language);
           // flag in case the locale currency isn't enabled
-          locale.currencyEnabled = locale.currencyEnabled;
+          locale.currencyEnabled = locale.currency.enabled;
```

Comment:
I feel like @jm-wanja just changed this to fix another bug. Can you check with her?

Reply:
sure

Reply:
@zenweasel  the reason for the change was because `locale` does not have a `currencyEnabled` field so it was always `undefined`. Just checked in with her and had her test if it affected the bug she worked on. She said it did not.